data/brands/amenity/restaurant.json

```
@@ -95,6 +95,19 @@
         "official_name": "Les 3 Brasseurs"
       }
     },
+    {
+      "displayName": "60secondstonapoli",
+      "id": "60secondstonapoli-051971",
+      "locationSet": {"include": ["de"]},
+      "tags": {
+        "amenity": "restaurant",
+        "brand": "60secondstonapoli",
```

It looks like this is branded as '60 seconds to napoli'

```suggestion
      "displayName": "60 seconds to napoli",
      "id": "60secondstonapoli-051971",
      "locationSet": {"include": ["de"]},
      "tags": {
        "amenity": "restaurant",
        "brand": "60 seconds to napoli",
```